hydromt/data_adapter.py

```diff
@@ -694,7 +694,17 @@ def round_latlon(ds, decimals=5):
     return ds
 
 
-PREPROCESSORS = {"round_latlon": round_latlon}
+def transpose_dims(ds):
+    x_dim = ds.raster.x_dim
+    y_dim = ds.raster.y_dim
+    ds = ds.transpose("time", y_dim, x_dim)
```

Comment:
I think we could do this in the RasterDataset.get_data(), but without specifying the name of the first dimension:

`ds = ds.transpose(..., y_dim, x_dim)`